src/components/v2/MyAccount/MyAccountStyles.ts

```diff
@@ -0,0 +1,105 @@
+import { css } from '@emotion/react';
+import { useTheme } from '@mui/material';
+
+export const useMyAccountStyles = () => {
+  const theme = useTheme();
+  return {
+    root: css`
+      font-size: 20px;
+      padding: ${theme.spacing(3)};
+    `,
+    row: css`
+      width: 100%;
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+    `,
+    apyWithXvs: css`
+      font-size: 14px;
```

Comment:
Why not use the `Typography` component with our theme variant to render this text?